mlflow/ml-package-versions.yml

```diff
@@ -660,7 +660,7 @@ langchain:
         pytest tests/langchain --ignore tests/langchain/test_langchain_autolog.py --ignore tests/langchain/test_langchain_evaluation.py
       fi
   autologging:
-    minimum: "0.1.0"
+    minimum: "0.0.335"
```

Comment:
Let's not update the version if the new version is older than the current version.

Reply:
updated